planning/behavior_velocity_planner/src/utilization/interpolate.cpp

```diff
@@ -12,15 +12,101 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <interpolation/spline_interpolation.hpp>
 #include <utilization/interpolate.hpp>
 #include <utilization/util.hpp>
 
+#include <autoware_auto_planning_msgs/msg/path_point_with_lane_id.hpp>
+#include <autoware_auto_planning_msgs/msg/path_with_lane_id.hpp>
+
 #include <vector>
 
 namespace behavior_velocity_planner
 {
 namespace interpolation
 {
+bool splineInterpolate(
```

Comment:
@taikitanaka3 I suggest putting this function in `utilization/path_utilization.cpp`. Then we can completely remove the `utilization/interpolate.hpp, .cpp` files with unused `LinearInterpolate` functions.